related.md

```diff
@@ -4,57 +4,268 @@ title: Related
 permalink: /related/
 ---
 
-# L4
+* TOC
+{:toc}
+
+# OSS Virtualization Projects
+---
+
+### ITL QubesOS
+
+> "... is a security-focused desktop operating system that aims to provide security through isolation. Virtualization is performed by Xen, and user environments are based on Fedora."
+
+- [History](http://en.wikipedia.org/wiki/Qubes_OS) &middot; [Funding](https://opencollective.com/qubes-os)
+- [Software](https://qubes-os.org/), *Poland*
+- [Research](https://www.qubes-os.org/research/) &middot; [Qubes Air](https://www.qubes-os.org/news/2018/01/22/qubes-air/) &middot; [Separation](https://invisiblethingslab.com/resources/2014/Software_compartmentalization_vs_physical_separation.pdf) &middot; [Intel x86](https://blog.invisiblethings.org/papers/2015/x86_harmful.pdf)
+- [Users](https://groups.google.com/forum/#!forum/qubes-users) &middot; [Developers](https://groups.google.com/forum/#!forum/qubes-devel) &middot; [News](https://www.qubes-os.org/news/) &middot; [SaltStack management](https://www.qubes-os.org/doc/salt/)
+
+### Bromium µ-Xen
+
+> micro-virtualization, Type-2 hypervisor, VM fast fork with copy-on-write memory/disk, deprivileged Windows host, display compositing, seamless user experience
+
+- [Software](https://bromium.com/opensource), *UK*
+- Presentations: [2012](http://www.slideshare.net/xen_com_mgr/xen-14203926) &middot; [2018](https://platformsecuritysummit.com/2018/speaker/pratt)
+- Linux port: [osresearch](https://github.com/osresearch/uxen)
+
+### Xen Servers
+
+- [XenServer / Citrix Hypervisor](http://xenserver.org) &middot; [source](https://www.citrix.com/community/citrix-developer/citrix-hypervisor1/), *China, UK*
+- [XCP-ng](https://xcp-ng.org), *France*
+- U.S. NRL Xenon Separation VMM: [2008](http://www-archive.xenproject.org/files/xensummitboston08/XenSummitSpring08.pdf) &middot; [2018](https://www.platformsecuritysummit.com/2018/speaker/kang/PSEC2018-Xenon-Separation-VMM-Myong-Kang.pdf), *USA*
+
+### AIS Redfield
+
+- [Bareflank](https://bareflank.github.io/hypervisor/) hypervisor toolkit, *USA*
```

Comment:
Bareflank should probably be in its own category, it's unrelated to Redfield as far I know.

Reply:
I've renamed this category to "AIS Virtualization" and revised the text for the Redfield entry.